resources/multisite_drupal_standard.make

```diff
@@ -13,6 +13,7 @@ includes[] = "drupal-core.make"
 
 projects[admin_menu][subdir] = "contrib"
 projects[admin_menu][version] = "3.0-rc6"
+projects[admin_menu][patch][]  = admin_menu_above_globab-nept-2677-.patch
```

Comment:
Should point to the patches folder. Also 'globab' is a typo and the ticket number is 2667. Don't forget to update the name of the patch file if you fix it here.